src/targets/java/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = require('require-directory')(module);
+
+
```

Comment:
you should use the `.editeconfig` file to make sure your files are properly spaced....